foreman/foreman.spec

```diff
@@ -621,6 +641,10 @@ if [ ! -e %{_datadir}/%{name}/config/initializers/encryption_key.rb -a \
   ln -s %{_sysconfdir}/%{name}/encryption_key.rb %{_datadir}/%{name}/config/initializers/
 fi
 
+# copy pregenerated apipie cache files
```

Comment:
The idea was that plugin provides docs for resource only if it overrides the core. As it depednds on the core it should be loaded later and win. It is true that with multiple overriding plugins there no rule (is this situation covered on route level?) There might be also  problem with core update.

We can move the copying to the rake task perhaps. That would solve the problem with update and we have access to Rails there if we have to figure out the plugin ordering. What do you think?